build/Microsoft.DotNet.Cli.Monikers.props

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <SdkBrandName>.NET Core SDK 1.1.5</SdkBrandName>
+    <SdkBrandName>.NET Core SDK 1.1.6</SdkBrandName>
     <SharedFrameworkBrandName>Microsoft .NET Core 1.1.5 - Runtime</SharedFrameworkBrandName>
```

Comment:
@leecow Is the runtime changing to 1.1.6 for this?

Reply:
Runtime shouldn't change right? It was just aspnetcore and later that were rev'ed, I'll let lee confirm though.

Reply:
Correct, no runtime update for this.